datajoint/autopopulate.py

```diff
@@ -16,28 +17,27 @@ class AutoPopulate(metaclass=abc.ABCMeta):
     """
     AutoPopulate is a mixin class that adds the method populate() to a Relation class.
     Auto-populated relations must inherit from both Relation and AutoPopulate,
-    must define the property populated_from, and must define the callback method _make_tuples.
```

Comment:
Why are we changing this to matlab style? I liked the more verbose version `populated_from`.